pig/src/itest/java/org/elasticsearch/hadoop/integration/pig/AbstractPigExtraTests.java

```diff
@@ -120,10 +124,27 @@ public void testTupleSaving() throws Exception {
                 "answers = LOAD 'src/itest/resources/tuple.txt' using PigStorage(',') as (id:int, parentId:int, score:int);" +
                 "grouped = GROUP answers by id;" +
                 "ILLUSTRATE grouped;" +
-                "STORE grouped into 'pig-test-tuple-structure/data' using org.elasticsearch.hadoop.pig.EsStorage('es.mapping.pig.tuple.use.field.names = true');";
+                "STORE grouped into '"+resource("pig-test-tuple-structure", "data")+"' using org.elasticsearch.hadoop.pig.EsStorage('es.mapping.pig.tuple.use.field.names = true');";
```

Comment:
nit: String.format may allow this read better ? (opinion)

Reply:
Should we just do that for all the Pig scripts? There are portions in other classes where multiple lines are injected using string concat, and replacing it with "%s" in those spots doesn't feel much cleaner...

Java needs string interpolation.

Reply:
nah, just wanted to ask the question if you thought it would read better. it's good as-is